source/whats-new.txt

```diff
@@ -22,6 +23,28 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-4.7:
+
+What's New in 4.7
+-----------------
+
+New features of the 4.7 {+driver-short+} release include:
+
+- Running ``MongoClient.connect()`` is now optional when connecting to your MongoDB instance
+- Use Zstandard compression when communicating with your MongoDB instance
+- Add support for the ``maxConnecting`` connection option
+- Change stream documents now show your updated documents before and after MongoDB applied changes to them
+- Added support for new change stream fields related to cluster to cluster replication
+- The ``estimatedDocumentCount`` method now uses the ``count`` database command
+- Deprecates the ``ResumeOptions`` interface
```

Comment:
Issue: Looking at the format of deprecations in this docs set, I think this should be called out at the very top in an important admonition (like how it was called out in the 4.0 notes).